starter_code/script.js

```diff
@@ -1,9 +1,100 @@
 window.onload = function() {
   document.getElementById("start-button").onclick = function() {
     startGame();
+
   };
 
   function startGame() {
 
+    var canvas = document.getElementById('road');
+    var ctx = canvas.getContext('2d');
+    drawRoad();
+    draw(car);
+
+
+
+    document.onkeydown = function(e) {
+    switch (e.keyCode) {
+    case 37: car.moveLeft();  console.log('left',  car); break;
+    case 39: car.moveRight(); console.log('right', car); break;
   }
+  updateCanvas();
+};
+
+function updateCanvas() {
+  ctx.clearRect(0,0,1500,1700);
+  ctx.fillText("car_x: " + car.x, 580,40);
+  ctx.fillText("car_y: " + car.y, 580,60);
+  draw(car);
+}
+
+updateCanvas();
+
+  };
+
+
+};
+
+
+
+    function draw(car) {
+      var car = {
+        x: 200,
+        y: 200,
+        moveLeft:  function() { this.x -= 25 ;},
+        moveRight: function() { this.x += 25 ;},
+  };
+
+    var img = new Image();
+    imgScale = 640/480;
+    img.onload = function() {
+     ctx.drawImage(img, car.x, car.y, 30*imgScale,80);
+      };
```

Comment:
Aquí el this no te funciona ya que estás con una función dentro de un método y pierdes el scope. Podrías usar un el famoso bind()!